gradle.properties

```diff
@@ -1,13 +1,10 @@
-# Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
-
 # Fabric Properties
-	minecraft_version=b1.7.3
-	loader_version=1.0.0
-	plasma_build=18
-	api_version=1.1.0
-
+minecraftVersion=b1.7.3
+loaderVersion=1.0.0
+plasmaBuild=18
+apiVersion=1.1.0
 # Mod Properties
-	mod_version = 1.0.0
-	maven_group = io.github.minecraftcursedlegacy.example
-	archives_base_name = cursed-legacy-example-mod
+modVersion = 1.0.0
+mavenGroup = io.github.minecraftcursedlegacy.example
+archivesBaseNameTwo = cursed-legacy-example-mod
```

Comment:
Also why “two”

Reply:
because if you look at the build.gradle.kts making it archives_base_name would be the wrong naming convention and archivesBaseName = archivesBaseName wouldn't work. I'll rename it to realArchivesBaseNameThough.

Reply:
this is stupid - there should only be one archives base name.